src/api/db/structure.sql

```diff
@@ -1435,6 +1435,7 @@ INSERT INTO `schema_migrations` (version) VALUES
 ('20181113095753'),
 ('20181201065026'),
 ('20190111130416'),
-('20190115131711');
+('20190115131711'),
+('20190130115727');
```

Comment:
Data migrations do not write into the `schema_migrations` table, they write to `data_migrations` (which we do not track in the schema as it's instance specific). Just get rid of this chunk.